lib/Backend/Commands.hs

```diff
@@ -91,7 +91,19 @@ createCmd (CreateOptions entryPath _edit force tags fields privateFields) = do
       & E.fields .~ HashMap.fromList allFields
       & E.tags .~ tags
 
-  -- TODO: check that a directory does not exist at the given path.
+    checkForEntriesInEntryPath :: EntryPath -> Sem r ()
+    checkForEntriesInEntryPath entryPath = do
+      let eParentDirs = forM ((NE.tail . Path.entryPathParentDirs) entryPath) Path.pathAsEntryPath
```

Comment:
Just a suggestion: maybe rename this variable to `parentDirsExceptRoot` to make it explicit that we're checking every parent dir path except the root path?

Reply:
Done